test/common/common/packed_struct_test.cc

```diff
@@ -65,7 +65,7 @@ TEST_F(PackedStructTest, StringStructMove) {
   EXPECT_FALSE(redirect_strings.get<RedirectStringElement::HostRedirect>().has_value());
 
   // Invoke move constructor.
-  RedirectStringsPackedStruct redirect_strings2(move(redirect_strings));
+  RedirectStringsPackedStruct redirect_strings2(absl::move(redirect_strings));
```

Comment:
I think we typically use std::move(). Is absl::move() different?

Reply:
I think it should be the same

Reply:
Ah, ok. in that case, let's use std::move() to be consistent.